lib/pages/settings/view/settings_page.dart

```diff
@@ -115,6 +115,20 @@ class _SettingsPageState extends State<SettingsPage> {
                     S.current.infoReadThePolicy(Utils.packageInfo.appName),
                   ),
                 ),
+                Visibility(
+                  visible: authProvider.currentUserFromCache.isAdmin == true,
+                  child: ListTile(
+                    key: const Key('HandleRequests'),
```

Comment:
This doesn't really matter, but why not 'AdminPanel'? It's what this button represents after all.